projects/addon-charts/components/pie-chart/pie-chart.component.ts

```diff
@@ -132,7 +132,7 @@ export class TuiPieChart {
     }
 
     private getDeg(value: number): number {
-        return 360 * (value / this.getSum(this.value));
+        return 360 * (value / this.getSum(this.value) ?? 0);
```

Comment:
I'm not sure I understand how that helps, the value on the left is never `null` or `undefined`, why does `??` change anything? 🤔

Reply:
but it's NaN, when we try to divide by zero(getSum for array like this [0, 0, 0] will be zero)
So we handle NaN and use "0" instead

Reply:
When we divide by zero we get `Infinity`, not `NaN` and `??` doesn't work with `NaN`:
![image](https://github.com/user-attachments/assets/977b429c-7293-4cf6-909e-b257f1c8de83)

Reply:
yeah, you are right
sorry for misunderstanding, i mean, if we divide zero by zero, it's only the case when it's `NaN`
but anyway i should use `||` instead of `??`
i'll fix it now

Reply:
but i change it to ternary, i guess it's more clean way
but u can correct me

Reply:
No need for ternary, just add `|| 0` at the very end.

Reply:
just thought about prevent even divide, but i guess it's unnecessary
fix it, so